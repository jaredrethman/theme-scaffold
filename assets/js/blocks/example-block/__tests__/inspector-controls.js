/**
 * Example Component - Tests.
 *
 * @package TenUpScaffold
 */

/**
 * Dependencies
 */
// NPM Modules.
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
// Components
import ExampleBlockInspectorControls from '../inspector-controls';

/**
 * SnapShot test
 */
describe( '<ExampleBlockInspectorControls />', () => {
	/**
	 * Ensure our Snapshot match's up
	 */
	describe( 'snapshot', () => {
		test( 'passed', () => {
			const snapShotTree = renderer.create(
				<ExampleBlockInspectorControls setAttributes={() => {}} />
			).toJSON();
			expect( snapShotTree ).toMatchSnapshot();
		} );
	} );

	/**
	 * Enzyme tests
	 *
	 * @link https://github.com/FormidableLabs/enzyme-matchers/tree/master/packages/jest-enzyme
	 */
	describe( 'enzyme', () => {
		const mountWrapper = shallow( <ExampleBlockInspectorControls setAttributes={() => {}} /> );
		test( 'exists', () => {
			expect( mountWrapper ).toExist();
		} );
	} );
} );
