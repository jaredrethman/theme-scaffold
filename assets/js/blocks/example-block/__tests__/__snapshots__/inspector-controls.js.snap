// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ExampleBlockInspectorControls /> snapshot passed 1`] = `null`;
